TextStyling-Kotlin/app/src/test/java/com/android/example/text/styling/parser/ParserTest.kt

```diff
@@ -23,40 +23,42 @@ import org.junit.Test
  */
 class ParserTest {
 
+    private val separator = System.getProperty("line.separator")
```

Comment:
make this const and rename to LINE_SEPARATOR

Reply:
Refactoring is done, but I am getting complaints from Kotlin about making LINE_SEPARATOR a const :-)